storage-proofs/src/porep/drg/circuit.rs

```diff
@@ -46,29 +40,23 @@ use crate::util::bytes_into_boolean_vec_be;
 /// * `data_root` - The merkle root of the data.
 /// * `replica_id` - The id of the replica.
 ///
-//implement_drgporep!(
-//    DrgPoRepCircuit,
-//    DrgPoRepCompound,
-//    "drg-proof-of-replication",
-//    false
-//);
 
 pub struct DrgPoRepCircuit<'a, H: Hasher> {
-    params: &'a <Bls12 as JubjubEngine>::Params,
-    replica_nodes: Vec<Option<Fr>>,
+    pub params: &'a <Bls12 as JubjubEngine>::Params,
```

Comment:
Do these fields need to be pub?

Reply:
brings them in line with the rest, the technically only need to be `pub(crate)`